aya/src/bpf.rs

```diff
@@ -717,19 +746,9 @@ impl Bpf {
     ///
     /// # Errors
     ///
-    /// Returns [`MapError::MapNotFound`] if the map does not exist. If the map is already borrowed
-    /// mutably with [map_mut](Self::map_mut) then [`MapError::BorrowError`] is returned.
-    pub fn map(&self, name: &str) -> Result<MapRef, MapError> {
-        self.maps
-            .get(name)
-            .ok_or_else(|| MapError::MapNotFound {
-                name: name.to_owned(),
-            })
-            .and_then(|lock| {
-                lock.try_read().map_err(|_| MapError::BorrowError {
-                    name: name.to_owned(),
-                })
-            })
+    /// Returns [`MapError::MapNotFound`] if the map does not exist.
```

Comment:
looks like the errors section can be removed now, and MapError::MapNotFound can probably be removed too now

Reply:
Arg sorry I missed that :( 

Done!